src/req_wrap-inl.h

```diff
@@ -20,12 +20,12 @@ ReqWrap<T>::ReqWrap(Environment* env,
                     AsyncWrap::ProviderType provider)
     : AsyncWrap(env, object, provider),
       ReqWrapBase(env) {
+  MakeWeak();
```

Comment:
I’d recommend to split this out into a separate PR. You might not want to carry this along the entire time you’re working on this PR, and it’s a good and meaningful change on its own.

Reply:
https://github.com/nodejs/node/pull/44074